### PR TITLE
Issue #42 Addressed

### PR DESCRIPTION
**Pull request by chb0github**
Waiting to hear from others. What resource does the drop-wizard example respond on? 

This appears to cover all cases but I need to be sure. Solving #42

### CODE REVIEW

File: katharsis-spring/src/main/java/io/katharsis/spring/boot/KatharsisRegistryConfiguration.java [hunk 1]
```diff
@@ -51,4 +52,23 @@ public ExceptionMapperRegistry exceptionMapperRegistry() throws Exception {
...
+    public ServiceUrlProvider getServiceUrlProvider() {
+        return new ServiceUrlProvider() {
+
+            @Value("${katharsis.pathPrefix}")
+            private String pathPrefix;
+
+            @Resource
+            private HttpServletRequest request;
+
+            public String getUrl() {
```
Comment (meshuga) [inline katharsis-spring/src/main/java/io/katharsis/spring/boot/KatharsisRegistryConfiguration.java]: MAJOR Add the "@Override" annotation above this method signature [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)

**Comment by meshuga** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: katharsis-spring/src/main/java/io/katharsis/spring/boot/KatharsisRegistryConfiguration.java (hunk 1 above)
Comment (chb0github) [inline katharsis-spring/src/main/java/io/katharsis/spring/boot/KatharsisRegistryConfiguration.java]: Easy enough. Thanks.

**Comment by meshuga** [comment]
Looks good for me, but I've been thinking if it would be suitable to refactor this, that is generalize a `ServiceUrlProvider` and `RequestContextParameterProvider` into a generic `Provider`, but on the other hand the second interface has two methods.

**Comment by chb0github** [comment]
Ok, I am going to run one final set of tests tomorrow morning and push. When is our next release?

**Comment by meshuga** [comment]
I think this Sunday will be good. Hope the weather will be bad ;)

**Pull request merged**